src/pykeen/utils.py

```diff
@@ -1777,6 +1780,63 @@ def isin_many_dim(elements: torch.Tensor, test_elements: torch.Tensor, dim: int
     return counts[inverse[: elements.shape[dim]]] > 1
 
 
+def minmax_normalize(array: np.ndarray) -> np.ndarray:
+    """
+    Normalize the given array using min-max normalization.
+
+    :param array:
+        array to normalize
+
+    :return:
+        normalized array
+    """
+    max_lit, min_lit = np.max(array, axis=0), np.min(array, axis=0)
+    return (array - min_lit) / (max_lit - min_lit + 1e-8)
+
+
+def filter_triples_by_relations(
+    triples: LabeledTriples, min_occurrences: int = 5, regex: str = r"^(?!.*http://rdf.freebase.com/key/).*$"
```

Comment:
make `min_occurrences` or `regex` optional?

Reply:
I changed the default values so that no filtering happens of they are not set by the user. Also, I set them explicitly for LiteralE in the experiment config files.